Source/Core/InputCommon/ControllerInterface/evdev/evdev.cpp

```diff
@@ -172,29 +184,26 @@ void Shutdown()
   StopHotplugThread();
 }
 
-evdevDevice::evdevDevice(const std::string& devnode) : m_devfile(devnode)
+evdevDevice::evdevDevice(const std::string& devnode) : m_devfile(devnode), m_interesting()
```

Comment:
You could just initialize `m_interresting` in at the declaration instead of here in the ctor.

Reply:
Will do.